docs/sources/operations/observability.md

```diff
@@ -82,6 +82,13 @@ exposed by Promtail at its `/metrics` endpoint. See Promtail's documentation on
 An example Grafana dashboard was built by the community and is available as
 dashboard [10004](https://grafana.com/dashboards/10004).
 
+## Metrics cardinality
+
+Some of the Loki observability metrics are emitted per tracked file (active), with the file path included in labels. 
```

Comment:
We should mention Promtail here explicitly... "per tracked file" only makes sense in the context of Promtail.

Removed redundancy:

```suggestion
Some of the Loki metrics are emitted per tracked file (active), with the file path included in labels. 
```

A hint about _which_ metrics are emitted per tracked file would be kind to the operator, so they are able to configure the scraping to avoid this issue.